src/routes/qr/+page.svelte

```diff
@@ -68,20 +62,4 @@ on the conference rolls.
 		display: grid;
 		gap: 4.5rem;
 	}
-
-	main :global .media svg {
-		width: 75%;
-		max-width: 18rem;
-	}
-
-	section {
-		grid-template-columns: repeat(auto-fit, minmax(21.5rem, 1fr));
-		gap: var(--space-xl);
-	}
-
-	@media (--viewport-md-up) {
-		main {
-			gap: 6.25rem;
-		}
-	}
```

Comment:
Page looks broken w/out this CSS.

Fix – use the new `ContentCardTemplate` / `ContentCardSection` components.

Reply:
yeah, actually i had it somewhere... maybe it got lost after solving the merge conflicts. Regardless — it will not take long